### PR TITLE
BAEL-102141 Review warnings in kotlin-build job

### PR DESCRIPTION
**Pull request by timis1**

### CODE REVIEW

File: kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt [hunk 1]
```diff
@@ -24,7 +24,7 @@ internal class FuelHttpLiveTest {
         val latch = CountDownLatch(1)
 
         "http://httpbin.org/get".httpGet().response{
-            request, response, result ->
+                _, response, result ->
```
Comment (alvin577austria) [inline kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt:27]: Can you fix the indentation here please.

File: kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt [hunk 2]
```diff
@@ -71,7 +71,7 @@ internal class FuelHttpLiveTest {
         val latch = CountDownLatch(1)
 
         Fuel.post("http://httpbin.org/post").response{
-            request, response, result ->
+                _, response, result ->
```
Comment (alvin577austria) [inline kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt:74]: Same here.

File: kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt [hunk 3]
```diff
@@ -149,10 +149,10 @@ internal class FuelHttpLiveTest {
     @Test
     fun whenDownloadFile_thenCreateFileResponseNotNullAndErrorNullAndStatusCode200() {
 
-        Fuel.download("http://httpbin.org/bytes/32768").fileDestination  { response, request ->
+        Fuel.download("http://httpbin.org/bytes/32768").fileDestination  { _, _ ->
             File.createTempFile("temp", ".tmp")
         }.response{
-            request, response, result ->
+                _, response, result ->
```
Comment (alvin577austria) [inline kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt:155]: And here.

File: spring-boot-kotlin/src/main/kotlin/com/baeldung/nonblockingcoroutines/handlers/ProductsHandler.kt [hunk 4]
```diff
@@ -3,10 +3,7 @@ package com.baeldung.nonblockingcoroutines.handlers
 import com.baeldung.nonblockingcoroutines.controller.ProductStockView
 import com.baeldung.nonblockingcoroutines.model.Product
 import com.baeldung.nonblockingcoroutines.repository.ProductRepositoryCoroutines
-import kotlinx.coroutines.Deferred
-import kotlinx.coroutines.FlowPreview
-import kotlinx.coroutines.GlobalScope
-import kotlinx.coroutines.async
+import kotlinx.coroutines.*
```
Comment (alvin577austria) [inline spring-boot-kotlin/src/main/kotlin/com/baeldung/nonblockingcoroutines/handlers/ProductsHandler.kt:6]: Please use individual imports.

File: kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt [hunk 5]
```diff
@@ -254,7 +254,7 @@ internal class FuelHttpLiveTest {
 
         Fuel.request(PostRoutingAPI.Post("1",null))
           .responseObject(Post.Deserializer()) {
-              request, response, result ->
+                  _, _, result ->
```
Comment (alvin577austria) [inline kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt:257]: And here.

**Review by alvin577austria — CHANGES_REQUESTED** [review]

File: kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt (hunk 2 above)
Comment (timis1) [reply kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt]: Done

File: kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt (hunk 3 above)
Comment (timis1) [reply kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt]: Done

File: spring-boot-kotlin/src/main/kotlin/com/baeldung/nonblockingcoroutines/handlers/ProductsHandler.kt (hunk 4 above)
Comment (timis1) [reply spring-boot-kotlin/src/main/kotlin/com/baeldung/nonblockingcoroutines/handlers/ProductsHandler.kt]: Done

File: kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt (hunk 5 above)
Comment (timis1) [reply kotlin-libraries-http/src/test/kotlin/com/baeldung/fuel/FuelHttpLiveTest.kt]: Done

**Review by alvin577austria — APPROVED** [review]

**Pull request merged**